config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp.yaml

```diff
@@ -48,7 +48,7 @@ spec:
            nova_api_network: internal_api
            nova_libvirt_network: internal_api
          edpm_chrony_ntp_servers:
```

Comment:
```suggestion
         timesync_ntp_servers:
```

Reply:
Right. The PR has been up for so long it has gone out of sync.